crates/turbopack-tests/tests/snapshot/minification/paren-remover/input/index.js

```diff
@@ -0,0 +1,30 @@
+function toFixed(value, maxDecimals, roundingFunction, optionals) {
```

Comment:
This test is copied from https://github.com/swc-project/swc/pull/8442

Reply:
Note: I also verified this fix by disabling `paren_remover` in the testing system for SWC minifier